src/components/card/card.component.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import './card.styles.css'
+
+import axios from 'axios'
```

Comment:
Add closing semi-colon to all import statements, setting up prettier will help you will all the formatting work that needs to be done.